logger/src/lib.rs

```diff
@@ -40,9 +41,30 @@ impl Logger {
                 std::env::temp_dir()
             } else {
                 let path = Path::new(&driver_path);
-                path.parent()
-                    .map(|p| p.parent().map(|p| p.join("logs")).unwrap())
-                    .unwrap_or_else(std::env::temp_dir)
+                // get the version number from the path
+                let version = path
+                    .parent()
+                    .map(|p| p.parent().unwrap())
+                    .unwrap()
+                    .file_name()
+                    .unwrap();
+                if let Some(user_dir) = UserDirs::new() {
+                    let log_dir = user_dir
+                        .document_dir()
+                        .map(|p| {
+                            p.join("MongoDB")
+                                .join("Atlas SQL ODBC")
+                                .join(version)
+                                .join("logs")
+                        })
+                        .unwrap_or_else(|| std::env::temp_dir());
+                    if !log_dir.exists() {
+                        std::fs::create_dir_all(&log_dir).unwrap();
```

Comment:
logging to the user's Documents location instead of where the driver is installed so we don't have to worry about updating our docs to show the user how to update the permissions on the directory, if they even can.

Reply:
It will write to the currently logged in user's documents directory, so not a shared directory among all users.

Reply:
This is correct. It's based on the current user not the user that installed it.